### PR TITLE
Quality of life improvements

### PR DESCRIPTION
**Pull request by mkrl**
Got rid of the most HTML in markdown, implemented theme-based notices and converted all existing notes to markdown.

- Previously appended with raw HTML `note [type]` classes have been replaced with [notices](https://github.com/mkrl/docs/blob/master/_sass/minimal-mistakes/_notices.scss) that are based on the Jekyll theme we're using (adopted for being used with its' dark variant). Those look way more pleasant and no longer look like they're out of place.
- Converted `` tags into markdown, got rid of centered images that stand out (let me know if that works).
- Added a tip on how to use notices into the doc page that explains the process of creating new pages.

Example screenshot of what changed

### CODE REVIEW

File: _posts/2019-08-27-guide_basic_zoning.md [hunk 1]
```diff
@@ -23,59 +23,53 @@ To follow this Guide you should:
 ## Opening the GUI
 In order to get the GUI to appear, type `mom_zone_showmenu` in the Console, it should look like this:  
 
-<img src="/assets/images/zone_guide/zone_menu.png" alt="Zone Menu" style="display: block; margin: auto;">
+![/assets/images/zone_guide/3d_cursor.png](3D Cursor)
```
Comment (Gocnak) [inline _posts/2019-08-27-guide_basic_zoning.md]: Wrong image

**Comment by Gocnak** [comment]
Hey sorry for the delay, been on vacation. Can you rebase onto latest master? It's good to go after that.

**Comment by braem** [comment]
This PR is quite stale. Any plans on rebasing it? We can always pull the work in if not.

**Comment by mkrl** [comment]
Dang, it's been a good year! I have completely missed the comment.
Yes, I will rebase this this or tomorrow afternoon.

**Comment by braem** [comment]
As we're approaching the 1 year mark of being open I will say that if you are too busy I can make the changes and co-author you for the work you've done.

**Comment by mkrl** [comment]
Will reopen with less garbage.

**Pull request closed without merging**